src/main/java/com/cburch/logisim/vhdl/gui/HdlContentEditor.java

```diff
@@ -249,8 +250,10 @@ private void configure(Project proj, HdlModel model) {
     close.addActionListener(frameListener);
     validate.addActionListener(frameListener);
 
+    AbstractTokenMakerFactory atmf = (AbstractTokenMakerFactory)TokenMakerFactory.getDefaultInstance();
+    atmf.putMapping("text/vhdl","com.cburch.logisim.vhdl.syntax.VhdlSyntax");
```

Comment:
I suggest to add a space after the comma following the first argument.

Reply:
Space added in following commit. Thank you, @maehne.